gamestonk_terminal/config_terminal.py

```diff
@@ -15,6 +15,30 @@
 # By default the jupyter notebook will be run on port 8888
 PAPERMILL_NOTEBOOK_REPORT_PORT = "8888"
 
+# Logging settings
+
+# 0 - INFO
```

Comment:
Instead of magic numbers could we have these as consts? Something like?
```
LOGGING_INFO = 0
LOGGING_DBG_TERMINAL_INFO_LIBS = 1
LOGGING_DBG_TERMINAL_DBG_LIBS = 2
```

Reply:
1. We don't want to introduce too many variables
2. Levels are ordered DEBUG+LIB > DEBUG > INFO. We want to follow logging convention - https://docs.python.org/3/howto/logging.html#logging-levels.